physics/GFS_PBL_generic.F90

```diff
@@ -591,7 +591,7 @@ subroutine GFS_PBL_generic_post_run (im, levs, nvdiff, ntrac,
   !    &     dtf,' kdt=',kdt,' lat=',lat
   !       endif
 
-        if (ldiag3d .and. flag_for_pbl_generic_tend .and. lssav) then
+        if_diag: if (ldiag3d .and. flag_for_pbl_generic_tend .and. lssav) then
```

Comment:
Can you please remove the if_diag: label here and in line 618, or do it consistently with if_lssav?

Reply:
I have added block labels to other large blocks in the subroutine.